populus/chain.py

```diff
@@ -714,7 +714,10 @@ def __enter__(self, *args, **kwargs):
         self.stack.enter_context(self.geth)
 
         if self.geth.is_mining:
-            self.geth.wait_for_dag(600)
+            # On Amazon EC2 2 VCPU small unit you get 32% of DAG generated
+            # within 600 seconds. Estimate is
+            # we can generate whole DAG in 40 minutes.
+            self.geth.wait_for_dag(60*40)
```

Comment:
I'll see if I can make all timeouts settable all the way through the stack, so there wouldn't any sort of inversion of control issues.